pkg/config/config.go

```diff
@@ -33,8 +33,9 @@ type AdditionalDirectory struct {
 
 // Finch represents the configuration file for Finch CLI.
 type Finch struct {
-	CPUs   *int    `yaml:"cpus"`
-	Memory *string `yaml:"memory"`
+	CPUs        *int    `yaml:"cpus"`
+	Memory      *string `yaml:"memory"`
+	CredsHelper *string `yaml:"credsHelper,omitempty"`
```

Comment:
Should probably make this a slice here since a user can install multiple credential helpers (although we only support ecr credentials helper with this change) and modify the Install [method](https://github.com/runfinch/finch/pull/462/files#diff-7ac37c68742ad8a4a56915628dcc400e073c8953e175af08b4ba4a63370a54cb).

Reply:
What is the resolution of this conversatino?

Reply:
Out of scope for current PR, this can be added when multiple cred helpers are supported.

Reply:
Wouldn't it be a breaking change to add multiple cred helpers late for renaming and changing data types?

Reply:
This is true, if preferred I can add this change. However I believe that adding this change can give the impression to the user that multiple cred helpers are supported even though this isn't the case.

Reply:
I think it is pretty common to keep the door open for future potential changes.

For example:
In Docker API, `/containers/json` always returns one name in `Names` field. https://docs.docker.com/engine/api/v1.43/#tag/Container

In ECR Public, `DescribeRegistries` always returns one registry. https://docs.aws.amazon.com/AmazonECRPublic/latest/APIReference/API_DescribeRegistries.html

Now seems we are believing there is a large chance to support multiple cred helpers later (unless we are not). We can't just keep the door close and leave the breaking change happen in the future.